templates/btax/includes/results/elasticity_table.html

```diff
@@ -0,0 +1,61 @@
+{% load humanize %}
```

Comment:
Does this need to be here? Is it the same as `templates/taxbrain/includes/results/elasticity_table.html`?